etna/pipeline/base.py

```diff
@@ -201,12 +199,6 @@ def validate_on_dataset(self, ts: TSDataset, horizon: int):
             diff = set(self.target_timestamps).difference(set(timestamps))
             raise ValueError(f"Some target timestamps aren't present in a given dataset: {reprlib.repr(diff)}")
 
-        dataset_description = ts.describe()
-
-        dataset_min_last_timestamp = dataset_description["end_timestamp"].min()
```

Comment:
Why did you decide to remove this check entirely? In the issue we discussed another solution.

Reply:
If last train timestamp is > `ts.index[-1]` then line 196 will be activated. So this check becomes redundant.